taw/__init__.py

```diff
@@ -50,5 +51,24 @@ def home():
                     )
             rows = sorted(rows, key=lambda row: row["player_1"].lower())
             return render_template("pairings.html", rows=rows, **ctx)
+        elif request.form["action"] == "match_slips":
+            pairings = form.parsed_pairings
+
+            rows = []
+            for pairing in pairings:
+                # We want to display each pairing twice, once for each player
+                rows.append(
+                    {
+                        "table_number": pairing.number,
+                        "player_1": pairing.player_1.name,
+                        "player_1_points": pairing.player_1.points,
+                        "player_2": pairing.player_2.name,
+                        "player_2_points": pairing.player_2.points,
+                    },
+                )
+            rows = sorted(rows, key=lambda row: row["player_1"].lower())
+            now = datetime.now()
+            date = now.strftime("%d/%m/%Y")
```

Comment:
1. There's a `datetime.date` library which lets you manipulate dates without time, which is probably what you want to do here
2. Because of this, `date` is more or less a reserved name :>

Something like this should work:
```suggestion
            from datetime import date
            today_str = date.today().strftime("%d/%m/%Y")
```